config/locales/views/en.yml

```diff
@@ -585,6 +585,13 @@ en:
       contact_support: contact Support
       last_renamed: You last changed your user name on %{renamed_at}.
       more_info: For information on how changing your user name will affect your account, please check out the %{account_faq_link}. Note that changes to your user name may take several days or longer to appear. If you are still seeing your old user name on your works, bookmarks, series, or collections after a week, please %{contact_support_link}.
+    confirmation:
+      contact_support: contact Support
+      go_back: Return to Archive front page
+      must_activate_html: "<strong>Important!</strong> You must activate your account within %{days_before_purge} days, or your account will expire. If this happens, you can sign up again using the same invitation."
```

Comment:
Thanks for shifting these all to use `I18n.t`! I think it would be helpful to have this one setup to handle pluralization as well, similar to https://github.com/otwcode/otwarchive/blob/master/config/locales/views/en.yml#L301